### PR TITLE
mpi PG: mpich has gccdevel now

### PR DESCRIPTION
**Pull request by eborisch**
Addresses https://trac.macports.org/ticket/62331

### CODE REVIEW

File: _resources/port1.0/group/mpi-1.0.tcl [hunk 1]
```diff
@@ -309,11 +309,14 @@ proc mpi.setup {args} {
         if {${os.major} <= 12 && $is_mpich} {
             lappend disabled -clang60 -clang70 -clang80 -clang90
         }
+        if {!$is_mpich} {
```
Comment (jmroot) [inline _resources/port1.0/group/mpi-1.0.tcl:312]: Should add `|| ${os.major} < 10` to the condition, to match the mpich portfile.

**Review by jmroot — CHANGES_REQUESTED** [review]
I noticed another issue: -gcc10 should also be appended on line 305.

**Review by tommyjons — APPROVED** [review]

File: _resources/port1.0/group/mpi-1.0.tcl (hunk 1 above)
Comment (yuu1111yuu) [reply _resources/port1.0/group/mpi-1.0.tcl:312]: $ ssh-keygen -t ed25519 -C "your_email@example.com"

**Review by yuu1111yuu — COMMENTED** [review]
$ ssh-keygen -t ed25519 -C "your_email@example.com"

**Comment by yuu1111yuu** [comment]
$ ssh-keygen -t ed25519 -C "your_email@example.com"

**Pull request closed without merging**